lib/pages/chat/input_bar/input_bar_shortcut.dart

```diff
@@ -37,6 +42,10 @@ class InputBarShortcuts extends StatelessWidget {
         ): () {
           _debouncer.run(() {
             controller?.addNewLine();
+            Timer(const Duration(milliseconds: 50), () {
+              scrollController
+                  .jumpTo(scrollController.position.maxScrollExtent);
+            });
```

Comment:
how about if user typing very quick?

Reply:
When user typing very fast (in my case, i long press it). it has no delay when scroll to bottom
https://github.com/linagora/twake-on-matrix/assets/43041967/0f58033b-e087-4216-a028-1ff4c1885bae